### ISSUE TITLE
'Random' endpoint consistently returning duplicate IDs

### ISSUE THREAD

**Issue by Tkwon123** [Giphy/GiphyAPI#44]
While providing a common search tag such as 'funny' in the random endpoint, I am consistently receiving duplicate ID values from the API. 

According to the website, there are > 30,000 tagged gifs, but in my tests, I am receiving ~20-30% duplicate values in each series of 100 requests. 

I noticed most of the gifs received are on the frontpage of the search results on the website; is this by design?

**Comment by namslam** [comment]
Hi @Tkwon123,

Thanks for the feedback. Currently the

**Comment by namslam** [comment]
Hi @Tkwon123,

Thanks for the feedback! Right now the random endpoint is pulling from ~10K of the top results from the query. Can you send me the request you're making so I can look into the dupes? Also it's worth noting that we have plans to upgrade the random endpoint in the upcoming months .

**Comment by Tkwon123** [comment]
Hey @namslam sorry for the late response. 

Seems pretty strange that of 10K results, I'm hitting a ~30% dupe rate, but not a big deal if there are upgrades incoming.

I just re-ran my script that builds an array of GIF ids by making requests at the random endpoint. Out of the 100 or so requests I made, I came out with the following dupes (27 out of 100 in this case): 

["JC41W0rJrmmTS","S2whkd4LyloiI","xWVS33PNW35y8","l41lUu4B3s5FwBxmM","xNhsQULL1B80o","phaN2NxXBzHMs","CJCNdPcgcuGRO","Kq3kAiCkJuFXO","4ECepQbplGd0I","ltB1n2fCwLRM4","xTiTnzEhdR9y9PNyc8","3o85xDk0GiBQjNl8OY","FhOCTD5CzzAm4","xTiTnpm7I3EULzATF6","tdr6m1OVPIqAM","cX9EynLj6nvFe","xTiTntziEFBzgPXqFi","3oEduHsO0taErvfz5C","3oEduYb9V8GjUH6O88","amBQZ8LUYpXqg","zTfwFhYUrIrnO","OpvoAdhfHlVde","XznaXMjKpSx2w","112VS6QqBw52Ks","3rgXBLS83pcYzv59Pq","Ld0P4AdeSDjoY","JftJQq8v6pGvu"]

Let me know if there's any other details youre interested in.
